### DISCUSSION TITLE
Could someone mentor me?

### DISCUSSION THREAD

**Discussion by zozo-yasuda** [mozilla/rhino#1459] [Q&A]
Hi, I'm a junior dev who uses Servoy developer, which uses Rhino as its javascript engine. I want to contribute, as well as learn more how the engine is implemented to gain a better understanding of the framework. Would anyone be willing to offer me some guidance? Even just a starting point like reading material would be greatly appreciated.

**Comment by p-bakker** [comment]
You can at least check out our documentation at rhino.github.io

Anything particular you're interested in contributing?

**Reply by zozo-yasuda** [reply]
Thanks for your response :) I'm interested in implementing later ES standards. Has there been a lot of activity in that area?

**Reply by p-bakker** [reply]
We're always working to become better compliant with the latest EcmaScript specification, but resources are sparse, so progress is slow :-)

Anything in particular ES thing you want to look into? There should be cases for most of the stuff that rhino isn't (properly) supporting right now.

Or you can have a look at https://github.com/mozilla/rhino/contribute for some cases that are deemed good starting points
